static/src/stylesheets/module/site-messages/_engagement-banner.scss

```diff
@@ -10,3 +10,11 @@
     overflow-x: hidden;
 }
 
+.site-message--banner.remote-banner-puzzles {
+    width: 100% !important;
+    z-index: 1090 !important;
+    background: none !important;
+    top: auto !important;
+    position: fixed !important;
+    max-height: 80vh;
+}
```

Comment:
Rather than duplicating most of the styles from the class above, perhaps we could have this be a modifier class instead?

```
.site-message--banner.remote-banner--puzzles {
	overflow: visible;
}
```

Then when rendering the banner we just do

```
container.classList.add('remote-banner', isPuzzlesBanner ? 'remote-banner--puzzles' : '');
```

Reply:
Nice! Thanks 👍